src/components/Cart/CartItem.js

```diff
@@ -1,85 +1,88 @@
+    CloseButton,
+    NumberInput,
+    NumberInputField,
+    NumberInputStepper,
+    NumberIncrementStepper,
+    NumberDecrementStepper,
 } from '@chakra-ui/react';
 import { CartContext } from '../../util/context';
-
 const CartItem = ({
-  imageURL,
-  altImage,
-  unitPrice,
-  title,
-  description,
-  id,
-  itemQuantity,
+    imageURL,
+    altImage,
+    unitPrice,
+    title,
+    description,
+    id,
+    itemQuantity,
 }) => {
-  const [quantity, setQuantity] = useState(itemQuantity);
-  const [totalprice, setTotalPrice] = useState(0);
-  const { handleAddQuantity, handleRemoveItemFromCart } =
-    useContext(CartContext);
+    const [quantity, setQuantity] = useState(itemQuantity);
+    const [totalprice, setTotalPrice] = useState(0);
+    const { handleAddQuantity, handleRemoveItemFromCart } =
+        useContext(CartContext);
 
-  const handleQuantityChange = (value) => {
-    setQuantity(parseInt(value));
-  };
+    const handleQuantityChange = (value) => {
+        setQuantity(parseInt(value));
+    };
 
-  useEffect(() => {
-    setTotalPrice((quantity * parseFloat(unitPrice)).toFixed(2));
-    handleAddQuantity(id, quantity);
-  }, [quantity]);
+    useEffect(() => {
+        setTotalPrice((quantity * parseFloat(unitPrice)).toFixed(2));
+        handleAddQuantity(id, quantity);
+    }, [quantity]);
 
-  return (
-    <Flex
-      maxW='full'
-      borderWidth='1px'
-      borderRadius='16px'
-      p={4}
-      mb={6}
-      align='center'
-    >
-      <Image
-        src={imageURL}
-        alt={altImage}
-        borderRadius='16px'
-        boxSize='150px'
-        objectFit='contain'
-      />
-      <Box p={4}>
-        <Heading size='l'>{title}</Heading>
-        <Text>{description}</Text>
-      </Box>
-      <Box p={4}>
-        <NumberInput
-          defaultValue={0}
-          min={1}
-          onChange={(value) => handleQuantityChange(value)}
-          value={quantity}
-          size='sm'
+    return (
+        <Flex
+            maxW='full'
+            borderWidth='1px'
+            borderRadius='16px'
+            p={4}
+            mb={6}
+            align='center'
         >
-          <NumberInputField maxW={32} />
-          <NumberInputStepper>
-            <NumberIncrementStepper />
-            <NumberDecrementStepper />
-          </NumberInputStepper>
-        </NumberInput>
-      </Box>
-      <Spacer />
-      <Flex p={4} align='center'>
-        <Text>{`$${totalprice}`}</Text>
-        <CloseButton onClick={() => handleRemoveItemFromCart(id)} m={2} />
-      </Flex>
-    </Flex>
-  );
+            <Image
+                src={imageURL}
+                fallbackSrc='https://via.placeholder.com/350'
+                alt={altImage}
+                borderRadius='16px'
+                boxSize='150px'
+                objectFit='contain'
+            />
+            <Box p={4}>
+                <Heading size='l'>{title}</Heading>
+                <Text maxW='400px'>{description}</Text>
```

Comment:
add `noOfLines={3}` or `noOfLines={4}` prop to limit number of lines. whatever looks better

Reply:
i will add noOfLines={4}. i didnt know about the prop. Thanks!